lib/options.js

```diff
@@ -19,6 +19,8 @@ const DEFAULT_OPTIONS = {
     secret: 'UNLEASH-SECRET',
     sessionAge: THIRTY_DAYS,
     adminAuthentication: 'unsecure',
+    importFromFile: null,
```

Comment:
Should be named `importFile` and default to `undefined`

Reply:
done